docs/guides/cloud-setup-invited.mdx

```diff
@@ -0,0 +1,220 @@
+   response = requests.post(url, headers=headers, data=data)
+
+   # Bearer token to authorize requests to the REST API
+   bearer_token = response.json()['access_token']
+   ```
+   </TabItem>
+</Tabs>
+
+### 2. Choose an authentication method
+
+Choose the appropriate authentication method, depending on your working environment:
+
+* [Create an environment variable for your API key (trusted Python environments)](#environment_variable)
+* [Use your API key directly (untrusted environment)](#rest-untrusted)
+
+<span id="environment_variable"></span>
+#### Create an environment variable (trusted environment)
+
+1.  To set the IQP_API_TOKEN environment variable in your system, you can add the following line to your shell profile (for example, .bashrc or .zshrc) or by setting it directly in your terminal:
+
+    ```shell
+    export IQP_API_TOKEN=<your-token>
+    ```
+    When you invoke the environment variable in your code, include `import os`, as in this example:
+
+    ```python
+    import os
+    api_token = os.environ['IQP_API_TOKEN']
+    ```
+
+    Note that when creating an environment variable, your API key is still stored locally in plain text, and should be safeguarded.
+1. Authenticate requests to the Qiskit Runtime REST API by including the CRN and bearer token to the request's headers.
+```bash
+curl -X 'GET' \
+   'https://quantum.cloud.ibm.com/api/v1/usage' \
+   '-H accept: application/json' \
+   '-H authorization: Bearer <BEARER_TOKEN>' \
+   '-H Service-CRN: crn:v1:bluemix:public:quantum-computing:us-east:a/b947c1c5a9378d64aed96696e4d76e8e:a3a7f181-35aa-42c8-94d6-7c8ed6e1a94b::'
+```
+
+<span id="rest-untrusted"></span>
+#### Use your API key directly (untrusted environment)
+
+Follow these steps if you must use an untrusted environment, such as a public computer.
+<Admonition type="caution">
+  When sharing code with others, ensure that your API key is not embedded directly within the Python script. Instead, share the script without the key and provide instructions for securely setting it up.
+
+  If you accidentally share your key with someone or include it in version control like Git, immediately revoke your key by following the [Deleting an API key](https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui#delete_user_key) topic on IBM Cloud.
+</Admonition>
+
+1. Authenticate requests to the Qiskit Runtime REST API by including the CRN and bearer token to the request's headers.
+
+  ```python
+# Refresh your API key on the dashboard after using the code as follows:
+api_key = "<your-key>"
+  ```
+1. Change your API key after each use by following the [Deleting an API key](https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui#delete_user_key) instructions on IBM Cloud.
+
+### 3. Optional: Configure your firewall
+
+    If necessary, use [this information](/docs/guides/quickstart-steps-org#firewall) to enable access to the IBM Quantum API endpoints.
```

Comment:
I would remove this from this guide, as an invited user you usually don't setup the firewall rules but the company which invited you to an account probably do for your environment.